packages/main/src/plugin/contribution-manager.ts

```diff
@@ -272,7 +272,18 @@ export class ContributionManager {
     try {
       jsonResultPs = JSON.parse(result.stdout);
     } catch (error) {
-      throw new Error(`unable to parse the result of the ps command ${result.stdout}`);
+      if (error instanceof SyntaxError) {
```

Comment:
@benoitf  Would simpler change, if code would check for '[' in the beginning of result.stdout and if it is missing fix stdout with
```
`[${result.stdout.replaceAll('\n', ',;')}]`
```
before parsing?
The